lib/sensu/spawn.rb

```diff
@@ -159,8 +159,13 @@ def child_process(command, options={})
         end
         [output, child.exit_code]
       rescue ChildProcess::TimeoutError
-        child.stop rescue nil
-        ["Execution timed out", 2]
+        output = "Execution timed out"
+        begin
+          child.stop
+        rescue => error
+          output += " - Unable to TERM/KILL the process: #{error}"
```

Comment:
can we enhance this to include the PID? I realize it would likely be in the stacktrace but for ease of parsing logs I think it makes sense to make it easier on the user. I'd guess it would be as simple as `output += " - Unable to TERM/KILL the process: #{child.pid}, stacktrace: #{error}`

Reply:
@majormoses excellent idea 👍 ill update to include child pid, but there is no (or useful) stacktrace for this error message.

Reply:
sounds good just replace `s/stacktrace/error_message` as `Operation not permitted` is useful enough I just was not sure if we wanted to include a full stacktrace if it provided more insight.